Library/Formula/sourcegraph.rb

```diff
@@ -0,0 +1,14 @@
+class Sourcegraph < Formula
+  desc "Smarter code host powering top-tier development teams"
```

Comment:
This sounds more like marketing-speech than an actual description of what the tool does. Can you improve this please?